### ISSUE TITLE
Automatically check for latest lmms version

### ISSUE THREAD

**Issue by zonkmachine** [LMMS/lmms#4131]
Make LMMS look for the latest release and inform the user if the package is outdated and no longer supported. This would help the user in trouble shooting and possibly take some load off the issue tracker/support channels.

Add an icon like :warning: with a descriptive text to the **About** window if there is a later release than the one used.

**Comment by tresf** [comment]
This was discussed quite a bit on the mailing list in in 2014.  I feel most of the points made are still valid.

http://linux-multimedia-studio-lmms.996328.n3.nabble.com/Roadmap-td6750.html

**Comment by zonkmachine** [comment]
Right. diizy suggested a `check for updates` button and that would work. Another way would be to just check if the date of the release if over something like one year old, and then suggest to (manually) look for updates.

Interacting with the net isn't my cup of tea but in my mind I would do something like:
 - Keap a list of the latest branches with the latest release in each of them.
 - Check against it once a month.

**Comment by PhysSong** [comment]
How about getting and parsing release information from [GitHub API](https://api.github.com/repos/LMMS/lmms/releases)? Though this will work only for tagged releases, it will be handy.

**Comment by tresf** [comment]
@PhysSong Sure however I think we should point at a domain that we own -- even if a simple DNS redirector -- so that we have full control in the event GitHub becomes unavailable (or more likely, undesired).
